app/controllers/exams_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ExamsController < ApplicationController
+    before_action :set_exam, only: [:show, :edit]
+  
+    def index
+      @exams = Exam.all
+    end
+  
+    def new
+      @exam = Exam.new
```

Comment:
@exam = Exam.new(subject_id: params[:subject_id]) It should be like this how will it work?